### DISCUSSION TITLE
Cant Find sharpmonoinjector.exe

### DISCUSSION THREAD

**Discussion by endivex** [winstxnhdw/lc-hax#25] [Q&A] [answered]
![image](https://github.com/winstxnhdw/lc-hax/assets/153818910/56e7e424-c2b6-40d1-9cb4-59e8e815fd30)

I already cloned the repository using the given code but after i tried to launch it still cant find sharpmonoinjector.exe

**Comment by winstxnhdw** [comment]
Send the full log when you do `requirements.bat`.

**Comment by KORVAAXX** [comment]
Hello, i have it too 
![issue](https://github.com/winstxnhdw/lc-hax/assets/154364126/586719a8-bdd8-4272-91f9-9b170094cfba)
this is the requirement.bat
and when the launch.bat:
![Capture](https://github.com/winstxnhdw/lc-hax/assets/154364126/d2ed4c49-ba86-4cac-91b8-bcff6ff4b4b9)

Sorry for my bad english as you can see, but in the error window it says : windows cannot find "./submodule.........."
check that you entered the correct name, then try again.

i'm new in this kind of cheats i don't know how to do, maybe drop a setup/instal video tutorial for all the not english persons.

**Comment by winstxnhdw** [comment]
Vous devez cloner récursivement le référentiel en utilisant l'indicateur `--recursive`

```bash
git clone --recursive https://github.com/winstxnhdw/lc-hax.git
```

**Comment by KORVAAXX** [comment]
that's what I did, I'll show you what that tells me:
![Capture](https://github.com/winstxnhdw/lc-hax/assets/154364126/8295841e-ed7e-4bcf-8b8c-71f567518542)

**Comment by winstxnhdw** [comment]
You should delete the original folder first before cloning again.

**Comment by KORVAAXX** [comment]
i delete lc-hax-main and clone it by the command ?

**Comment by winstxnhdw** [comment]
Delete `lc-hax`, yes.

**Comment by KORVAAXX** [comment]
i deleted the lc-hax and type git clone --recursive https://github.com/winstxnhdw/lc-hax.git in the cmd but :
![Capture](https://github.com/winstxnhdw/lc-hax/assets/154364126/45be379c-141f-4d0c-8be6-9d29c480e222)

**Comment by KORVAAXX** [comment]
add me on discord to explain me please xDD 
discord: korvaaxx

**Comment by winstxnhdw** [comment]
What did you do to get that message?

**Comment by KORVAAXX** [comment]
> Qu'avez-vous fait pour recevoir ce message ?

i just deleted the lc-hax and type git clone --recursive https://github.com/winstxnhdw/lc-hax.git in the CMD

**Comment by winstxnhdw** [comment]
Okay, now follow the steps in the [README](https://github.com/winstxnhdw/lc-hax#installation).

**Comment by KORVAAXX** [comment]
> Okay, now follow the steps in the [README](https://github.com/winstxnhdw/lc-hax#installation).

just to clarify i delete the entire lc-hax file?

**Comment by winstxnhdw** [comment]
Yes. The entire thing. Make sure you don't even see the folder `lc-hax`.

**Comment by KORVAAXX** [comment]
> Yes. The entire thing. Make sure you don't even see the folder `lc-hax`.

ok so that's what I did and after deleting and doing git clone --recursive https://github.com/winstxnhdw/lc-hax.git, I run the requirement.bat and it's done. puts message:

MSBuild version 17.8.3+195e7f5a3 for .NET
MSBUILD: error MSB1009: The project file does not exist.
Switch: lc-hax
MSBuild version 17.8.3+195e7f5a3 for .NET
MSBUILD: error MSB1003: Specify a project or solution file. The active working directory contains none.
Press any key to continue...
![Capture](https://github.com/winstxnhdw/lc-hax/assets/154364126/05134e84-53e8-4d31-ae6d-9746494f541d)

**Comment by KORVAAXX** [comment]
j

> > Yes. The entire thing. Make sure you don't even see the folder `lc-hax`.
> 
> ok so that's what I did and after deleting and doing git clone --recursive https://github.com/winstxnhdw/lc-hax.git, I run the requirement.bat and it's done. puts message:
> 
> MSBuild version 17.8.3+195e7f5a3 for .NET MSBUILD: error MSB1009: The project file does not exist. Switch: lc-hax MSBuild version 17.8.3+195e7f5a3 for .NET MSBUILD: error MSB1003: Specify a project or solution file. The active working directory contains none. Press any key to continue... ![Capture](https://private-user-images.githubusercontent.com/154364126/291680237-05134e84-53e8-4d31-ae6d-9746494f541d.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDMwMTA2OTcsIm5iZiI6MTcwMzAxMDM5NywicGF0aCI6Ii8xNTQzNjQxMjYvMjkxNjgwMjM3LTA1MTM0ZTg0LTUzZTgtNGQzMS1hZTZkLTk3NDY0OTRmNTQxZC5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIxOVQxODI2MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zODRiN2I1MTcwYmNjZTFhMmE2YmU0YWQ1OWM1N2E0YjVmYjg1MTI0MDUzZDRhMWE1NDAyNjM5OGFmZjVjNmVhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.h8JdNr6Es3ehGkcf6BwQ9B50Eo2Ok5iP-JuTnXZnO5A)

**Comment by winstxnhdw** [comment]
Can you look inside your submodules folder? Is SharpMonoInjectorCore there? You should see this.

![image](https://github.com/winstxnhdw/lc-hax/assets/56998716/d48e6a83-d75f-4ac3-a0a8-eccc46777fe2)

**Comment by KORVAAXX** [comment]
![Capture](https://github.com/winstxnhdw/lc-hax/assets/154364126/3aec5f63-45f6-44c8-b082-22092536c237)
no there is nothing

**Comment by winstxnhdw** [comment]
Hahaha, it is because you are not using the folder that you have cloned. Click on `requirements.bat` inside your `lc-hax` folder, not `lc-hax-main`.

**Comment by KORVAAXX** [comment]
i don't have requirements.bat inside my lc-hax folder 
![Capture](https://github.com/winstxnhdw/lc-hax/assets/154364126/5895a6fe-9765-4c83-bbd4-198e4c714e04)

**Comment by winstxnhdw** [comment, marked as answer]
You misunderstand.

I know what you did. You downloaded the repository as zip and extracted the files right? This is wrong. If you download as a zip, you will not download `SharpMonoInjectorCore`.

Instead, use the folder that you clone. When you clone a repository, a folder named `lc-hax` will be created.

**Comment by winstxnhdw** [comment]
Click on `Open Git Bash Here` and recursively clone the repository again. I think you can't find the folder because Git Bash defaults to `%USERPROFILE%`.

![image](https://github.com/winstxnhdw/lc-hax/assets/56998716/ec04074e-828c-4a37-ad5b-a9d3f804be68)

**Comment by KORVAAXX** [comment]
I don't understand, I can't, it seems simple but I'm useless xD I don't know how you're going to help me,

despite what you told me I don't see how to open with GitBashHere you can, maybe make a video tutorial or make me a list of what I have to do in order, your cheat is really interesting I want to trolling my friends xD

**Comment by winstxnhdw** [comment]
Okay, let's try this. Open Git Bash however you did before. Then type the following.

```ps1
explorer.exe  .
```

You should see a folder pop up. In this folder, you should also see a folder named `lc-hax`. From there, follow the instructions in the README.

**Comment by KORVAAXX** [comment]
> D'accord, essayez ça. Ouvrez Git Bash comme vous le feriez auparavant. Tapez ensuite ce qui suit.
> 
> ```powershell
> explorer.exe  .
> ```
> 
> Vous devriez voir un dossier apparaître. Dans ce dossier, vous devriez également voir un dossier nommé `lc-hax`. À partir de là, suivez les instructions du README.

When i open git bash and i type explorer.exe it opens me all my windows folders

**Comment by winstxnhdw** [comment]
Correct. Do you see `lc-hax` in there?

**Comment by KORVAAXX** [comment]
> us voyez `lc-hax`là-ded

no it opens me this folder 
![Capture](https://github.com/winstxnhdw/lc-hax/assets/154364126/9e1ebeb5-5e8e-48a2-a013-1c73125c6353)

**Comment by winstxnhdw** [comment]
You forgot to add `.`

It's `explorer.exe .`

**Comment by KORVAAXX** [comment]
i type explorer.exe in git bash cmd
![1](https://github.com/winstxnhdw/lc-hax/assets/154364126/7cf67599-fb5a-4750-8b5e-2148c651d410)
![2](https://github.com/winstxnhdw/lc-hax/assets/154364126/a980b78f-24d6-4389-9b36-977170c6cb21)

**Comment by winstxnhdw** [comment]
You forgot the period `.` again.

![image](https://github.com/winstxnhdw/lc-hax/assets/56998716/e655ac26-45b0-4850-a4b6-4b86c142aa99)

**Comment by KORVAAXX** [comment]
okay i see it, and now
![Capture](https://github.com/winstxnhdw/lc-hax/assets/154364126/fe995209-4660-46aa-9a3c-3115cc3e6201)

**Comment by winstxnhdw** [comment]
merci mon dieu..

**Comment by KORVAAXX** [comment]
> merci mon dieu..

Now that I found it, what should I do?

**Comment by winstxnhdw** [comment]
Follow the instructions in the [README](https://github.com/winstxnhdw/lc-hax#installation).

**Comment by KORVAAXX** [comment]
> Suivez les instructions du [fichier README](https://github.com/winstxnhdw/lc-hax#installation) .

ITS WORKIIIIIING !!!!, is there a activable menu or just commands ?

**Comment by winstxnhdw** [comment]
Congratulations. Just commands and keybinds.

**Comment by KORVAAXX** [comment]
> Toutes nos félicitations. Juste des commandes et des raccourcis clavier.

I'm sorry to have bothered you so much but I really wanted your mod, it's unique I've never seen such new options in the mod, thank you again and continue to do beautiful things that are at least evil ;)

good luck to you.

**Comment by rmgs6** [comment]
Hey. I ran the git bash and then pasted the first line of code. Then went to cmd and did explorer.exe  . And then clicked requirements. After I launched the game and the injector but it keeps saying I’m missing Sharpmonoinjectorcore. I tried deleting lc-hax folder and it also said permission denied when I tried to re clone. Not into coding or anything like this at all. I got the menu to work only the first time and now it keeps saying it’s missing the file even though I see it in the submodules folder. If someone can please explain how I can get it working again that would be amazing

**Reply by rmgs6** [reply]
I followed the readme file but everytime I get to the injection part it says it can’t find the file

**Reply by rmgs6** [reply]
Do I need to run everything in order every time I want to play? I tried to follow the instructions exactly and it keeps giving me an error and won’t inject

**Reply by rmgs6** [reply]
Also what is the command to make all of the furniture sideways?

**Reply by winstxnhdw** [reply]
Show me what it says when you execute `requirements.bat`.

**Reply by winstxnhdw** [reply]
No, after installation, you only need to use `launch.bat`.

**Reply by winstxnhdw** [reply]
`/ct`

**Reply by winstxnhdw** [reply]
A real image please.

**Reply by rmgs6** [reply]
![image](https://github.com/winstxnhdw/lc-hax/assets/161122447/b2ad6fa2-1597-434e-ae67-e1ad88ddc8c5)

**Reply by winstxnhdw** [reply]
Your antivirus is preventing you from building the injector.

**Reply by rmgs6** [reply]
How do I get around that

**Reply by rmgs6** [reply]
Just disable it? Or is there a way I can keep it on. It worked the first time

**Reply by winstxnhdw** [reply]
The last section in the README tells you how you can whitelist it.

**Reply by rmgs6** [reply]
When I paste it into the cmd I get this 
![image](https://github.com/winstxnhdw/lc-hax/assets/161122447/c1f78952-f782-40eb-8b3e-999bde57e583)

**Reply by rmgs6** [reply]
I don’t know how to get to the lc hax directory

**Reply by winstxnhdw** [reply]
`cd lc-hax`

**Reply by winstxnhdw** [reply]
You gotta use PowerShell with admin privileges though.

**Reply by rmgs6** [reply]
I ran as admin but it says it cannot find the specified path. I typed cd lc-hax

**Reply by winstxnhdw** [reply]
Oh right. In your case, you’d need to do

```
cd C:\Users\ryanm\lc-hax
```

**Reply by rmgs6** [reply]
![image](https://github.com/winstxnhdw/lc-hax/assets/161122447/8f35f911-7207-4269-9e1e-8f9d7d61739d)

**Reply by rmgs6** [reply]
Still saying the same thing

**Reply by rmgs6** [reply]
Sorry I am very bad with this stuff

**Reply by winstxnhdw** [reply]
That’s not PowerShell buddy

**Reply by rmgs6** [reply]
Haha my bad. Let me try now

**Reply by rmgs6** [reply]
It works. Thank you so much

**Reply by rmgs6** [reply]
I am trying to use /ct but it says target player is not alive or found.

**Reply by winstxnhdw** [reply]
Read the README. That command requires a player ID or a player name as an argument.

**Reply by rmgs6** [reply]
In the lobby it shows all names as player. I tried to copy the steam id too but it still says target player is not alive or found. I also tried their normal name

**Reply by winstxnhdw** [reply]
Steam ID? What does that have to do with the player ID? If you do `/players` or you have ESP on, you can see the player ID beside their name. In vanilla LC, it’s usually 0-4.

**Reply by rmgs6** [reply]
Can you make the terminal and all furniture sideways as well?

**Reply by winstxnhdw** [reply]
One day, probably. We are prioritising other features like possession first.

**Reply by rmgs6** [reply]
Gotcha. I’ve seen a mod menu where they can make all furniture sideways. Do you recognize it?

**Reply by winstxnhdw** [reply]
Nope.

**Reply by rmgs6** [reply]
How do I keep my normal name 😂😂

**Reply by winstxnhdw** [reply]
Disable anti-kick.

**Comment by rmgs6** [comment]
So I only need to run the injector when I launch now? Will I ever need to re clone or execute the requirements?

**Reply by winstxnhdw** [reply]
As long as the folder exists, you only need `launch.bat`. If you ever delete the folder, you’ll have to repeat the steps.

**Reply by rmgs6** [reply]
Thank you so much. You made my day!

**Comment by rmgs6** [comment]
Hey. I am not able to spawn mobs every time I try the name it says invalid mob. I am also not able to use the fatality command properly. Also for the possession mod how do I enable it and use it? Thank you !

**Reply by joep26020** [reply]
before you ask a question read the entire 'read.me'

**Reply by rmgs6** [reply]
I did

**Reply by rmgs6** [reply]
Some of the commands don’t work

**Reply by joep26020** [reply]
did you notice you have to be host to use certain commands? or that to use fatality there are certain parameters like which player and which monster to use?

**Reply by rmgs6** [reply]
Yes

**Reply by joep26020** [reply]
so you good now?

**Reply by rmgs6** [reply]
Not sure. I have tried to spawn mobs but the names don’t work. Even as host it won’t let me. That’s really the only thing not fully functional.

**Reply by joep26020** [reply]
have you tried this on vanilla?

**Reply by rmgs6** [reply]
I used the injector and am running the mod menu. Maybe some of the enemy spawn names are case sensitive or smth like that. An enemy list or smth like that would be good to add to the read.me

**Reply by joep26020** [reply]
check the file for the command. the list is probably there

**Reply by winstxnhdw** [reply]
Yeah, it’s not normal for it to say invalid enemy. We use fuzzy string matching, so there’s no need for you to write the exact names. If it says `invalid enemy`, that means your game doesn’t have the enemy prefabs for some reason.

For example, you can do:

```console
/spawn nutcracker 0 1
/spawn cracker 0 1
/spawn crackernut 0 1
```

The list for fatality is in the README.

**Reply by rmgs6** [reply]
Hey. Is the /ct command gone? i really liked being able to mess with the furniture.

**Reply by winstxnhdw** [reply]
Replaced with `/spin`
